src/mainwindow.cpp

```diff
@@ -855,6 +856,11 @@ void MainWindow::playPulse(qint64 msecsElapsed)
         ui->tableWidget->item(row, COLUMN_START)->setData(Qt::UserRole, progressionNext);
         ui->tableWidget->item(row, COLUMN_END)->setData(Qt::UserRole, progressionCurrent);
     }
+    logFile->open(QIODevice::WriteOnly | QIODevice::Append);
```

Comment:
This implementation has the upside that it will log stuff in both manual and automatic mode. However, since this function is called very often, it logs the same entry several times which could be problematic in terms of space. We should find a more suitable place to log the remaining time.